### ISSUE TITLE
Bootstrap Modal data-dismiss is not working

### ISSUE THREAD

**Issue by Jessy-developer** [puikinsh/ElaAdmin#5]
Bootstrap Modal data-dismiss is not working due sidebar menu

**Comment by consciousdreams** [comment]
Hi @Jessy-developer , 
I had the same issue, I solved the problem by moving sidebarmenu.js before bootstrap js.
Hope this helps you!

Francesco

**Comment by Jessy-developer** [comment]
Yeah!, After moving sidebarmenu.js above bootstrap.js will cause a side effect that side bar menu drop down is not working and many more side effects looks like something getting over ridden in because some common parameter in both the files

**Comment by consciousdreams** [comment]
I have updated the [MetisMenu](https://github.com/onokumus/metismenu) (that is the sidebar menu) to the most recent version and it seems to work fine.

Here is the CDN of the MetisMenu's latest version:
`<script src="https://cdnjs.cloudflare.com/ajax/libs/metisMenu/2.7.4/metisMenu.min.js" integrity="sha256-YqTSp9JlAIbtktRVVwWzOEXJAIs8vE/r79vR7jcvYj0=" crossorigin="anonymous"></script>`

**Comment by bruno-lombardi** [comment]
@consciousdreams your solution did work!!!! I was about to get crazy googling for "bootstrap modal don't work", and you got it! Thanks a lot. I haven't tested the sidebar yet, but I do think it should work fine.

**Comment by Jessy-developer** [comment]
@consciousdreams @bruno-lombardi I have Updated the  MetisMenu (that is the sidebar menu) to the most recent version but now the modal is flickering when it is opened and it is not getting closed when clicked on close button data-dismiss is not working.

**Comment by effetx** [comment]
Hello everybody,
possible quick solution is to remove "fade" class from modal, directly or following this jquery function:
$('.modal').removeClass('fade');

**Comment by FatenElhariry** [comment]
i have the same problem

**Comment by FatenElhariry** [comment]
when i remove class fade it works will @effetx  thanks for your solution

**Comment by bososha** [comment]
@Jessy-developer Hi I have the same problem and the preview modal of bootstrap-fileinput.js still cannot dismiss. Seems that the problem is due to the function transitionEndTest in sidebarmenu.js

**Comment by hellonts** [comment]
@effetx very good, thanks
